examples/datasets/plot_limo_data.py

```diff
@@ -33,146 +35,282 @@
+# Visualize effect of stimulus phase-coherence
+# --------------------------------------------
+#
+# Since phase-coherence
+# determined whether a face stimulus could be easily identified,
+# one could expect that faces with high phase-coherence should evoke stronger
+# activation patterns along occipital electrodes.
+
+phase_coh = limo_epochs.metadata['phase-coherence']
+# get levels of phase coherence
+levels = sorted(phase_coh.unique())
+# create labels for levels of phase coherence (i.e., 0 - 85%)
+labels = ["{0:.2f}".format(i) for i in np.arange(0., .90, .05)]
+
+# create dict of evokeds for each level of phase-coherence
+evokeds = {label: limo_epochs[phase_coh == level].average()
+           for level, label in zip(levels, labels)}
+
+# pick channel to plot
+electrodes = ['C22', 'B11']
+# create figures
+for electrode in electrodes:
+    fig, ax = plt.subplots(figsize=(8, 4))
+    plot_compare_evokeds(evokeds,
+                         axes=ax,
+                         ylim=dict(eeg=(-15, 15)),
+                         picks=electrode,
+                         cmap=("Phase coherence", "magma"))
+
+###############################################################################
+# As shown above, there are some considerable differences between the
+# activation patterns evoked by stimuli with low vs. high phase-coherence at
+# the chosen electrodes.
+#
+#
+# Prepare data for linear regression analysis
+# --------------------------------------------
+#
+# Before we test the significance of these differences using linear
+# regression, we'll interpolate missing channels that were
+# dropped during preprocessing of the data.
+# Furthermore, we'll drop the EOG channels (marked by the "EXG" prefix)
+# present in the data:
+
+limo_epochs.interpolate_bads(reset_bads=True)
+
+limo_epochs.drop_channels(['EXG1', 'EXG2', 'EXG3', 'EXG4'])
+
+###############################################################################
+# Define predictor variables and design matrix
+# --------------------------------------------
+#
+# To run the regression analysis,
+# we need to create a design matrix containing information about the
+# variables (i.e., predictors) we want to use for prediction of brain
+# activity patterns. For this purpose, we'll use the information we have in
+# ``limo_epochs.metadata``: phase-coherence and Face A vs. Face B.
+
+# name of predictors + intercept
+predictor_vars = ['face a - face b', 'phase-coherence', 'intercept']
+
+# create design matrix
+design = limo_epochs.metadata[['phase-coherence', 'face']].copy()
 design['face a - face b'] = np.where(design['face'] == 'A', 1, -1)
-names = ['intercept', 'face a - face b', 'phase-coherence']
+design['intercept'] = 1
+design = design[predictor_vars]
+
+###############################################################################
+# Now we can set up the linear model to be used in the analysis using
+# MNE's linear_regression function (see func:`mne.stats.linear_regression`).
```

Comment:
```suggestion
# MNE-Python's func:`~mne.stats.linear_regression` function.
```

Reply:
the `~` character tells Sphinx to omit the `mne.stats.` prefix when it renders the function name.

Reply:
ok cool, thanks, I'll keep this in mind in the future.